addon/components/forgot-password-form.js

```diff
@@ -0,0 +1,77 @@
+import Ember from 'ember';
+import layout from '../templates/components/forgot-password-form';
+import getConfig from 'ember-authenticate-me/configuration';
+
+export function forgotPassword(email) {
+  const emberAuthenticateMeConfig = getConfig();
+  const PASSWORD_RESET_URI = emberAuthenticateMeConfig.passwordResetUri ||
```

Comment:
Seems odd to use different name conventions for what appears to be similar const vars